.github/workflows/publish-snapshot.yml

```diff
@@ -110,7 +110,10 @@ jobs:
             -Dmaven.repo.local=.m2/repository \
             deploy \
             -DskipTests \
-            -PfastSkip,sign,pmd-release -Dcyclonedx.skip=false
+            -PfastSkip,sign,pmd-release \
+            -Dcyclonedx.skip=false \
+            -Dmaven.javadoc.skip=false \
+            -Dmaven.source.skip=false
```

Comment:
This is actually a fix that maybe should be extracted out of this PR, as it affects the snapshot builds.
Currently, no sources and javadoc artifacts are available at https://central.sonatype.com/service/rest/repository/browse/maven-snapshots/net/sourceforge/pmd/pmd-core/